### PR TITLE
Adding placeholder message for faws support

### PR DESCRIPTION
**Pull request by gabecase**
Reviewers can you help me come up with the best way to clarify that
- Support for federation and faws is currently not available
- We are planning on supporting it at some point

### CODE REVIEW

File: doc/attribmapping-basics/faws-mapping.rst [hunk 1]
```diff
@@ -1,50 +1,8 @@
 .. _faws-mapping-ug:
 
-Assigning Fanatical Support for AWS permissions
+Fanatical AWS Support
```
Comment (nicholasbwagner) [inline doc/attribmapping-basics/faws-mapping.rst]: The product name is "Fanatical Support for AWS".

Comment (gabecase) [reply doc/attribmapping-basics/faws-mapping.rst]: Roger will fix that

File: doc/attribmapping-basics/faws-mapping.rst [hunk 2]
```diff
@@ -1,50 +1,8 @@
...
-Contact Fanatical AWS support
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-After your |amp| has the correct section added, contact your **Fanatical
-Support for AWS** support or account team, and they will help you further
-configure the specific AWS permissions and roles needed for controlling
-federated user access.
+Federation support for Fanatical AWS is currently unavailable but is being
+worked on. Instructions for setting up federation for Fanatical AWS access
+will be available here once it's available to all customers.
```
Comment (nicholasbwagner) [inline doc/attribmapping-basics/faws-mapping.rst]: Using "Fanatical Support for AWS" is tricky, so I can't just find and replace in this section! How about something like this:

We are currently integrating Fanatical Support for AWS with Identity Federation. 

As soon as Identity Federation is available to Fanatical Support for AWS customers, we'll provide detailed instructions here about how to get set.

File: doc/attribmapping-basics/faws-mapping.rst [hunk 3]
```diff
@@ -1,50 +1,8 @@
...
-
-
-Contact Fanatical AWS support
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-After your |amp| has the correct section added, contact your **Fanatical
-Support for AWS** support or account team, and they will help you further
-configure the specific AWS permissions and roles needed for controlling
-federated user access.
+Federation for Fanatical Support for AWS is currently unavailable but is being
```
Comment (catlook) [inline doc/attribmapping-basics/faws-mapping.rst]: Change the lines to:  Federation for Fanatical Support for AWS is currently unavailable but is under development. Instructions for setting up Federation will be available after it is released to all customers.

**Pull request merged**